lib/constants.js

```diff
@@ -0,0 +1,247 @@
+export function getCoinIconUri(coinName) {
+    switch (coinName) {
```

Comment:
While this works for now, we will need to handle the edge case of when a url is bad or when the url change